src/main/kotlin/me/serce/solidity/ide/inspections/fixes/ImportFileAction.kt

```diff
@@ -127,7 +126,7 @@ class ImportFileAction(
 
     fun buildImportPath(source: VirtualFile, destination: VirtualFile): String {
       return Paths.get(source.path).parent.relativize(Paths.get(destination.path)).toString().let {
-        val separator = File.separator
+        val separator = "/"
```

Comment:
For my understanding, would this change be necessary considering the replace call? 

I haven't tested, but my understanding is that the hardcoded `/` here would break the `contains` calls below since they operate on relativized paths.